frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/columns/simple_column_block_component.dart

```diff
@@ -115,7 +116,11 @@ class SimpleColumnBlockComponentState extends State<SimpleColumnBlockComponent>
       crossAxisAlignment: CrossAxisAlignment.start,
       children: node.children.map(
         (e) {
-          Widget child = IntrinsicHeight(
+          Widget child = Provider.value(
```

Comment:
It's a newly created value `DatabasePluginWidgetBuilderSize`. I suppose we should use 'create' instead?